composer.json

```
@@ -15,8 +15,7 @@
     },
     "require-dev" : {
         "squizlabs/php_codesniffer" : "2.9.2",
-        "phpunit/phpunit" : "6.5.*",
-        "phpunit/dbunit": "3.0.*",
+        "phpunit/phpunit" : "^8",
         "facebook/webdriver" : "dev-master",
         "phpmd/phpmd": "^2.6",
         "phan/phan": ">1.2.2"
```

```suggestion
        "phan/phan": ">2.2.3"
```

Since we are updating `php-ast` in this PR, I think it makes sense to update phan to the latest version also